src/UartSerial.cpp

```diff
@@ -156,7 +157,7 @@ UartSerial::begin (
     // Leave all output flags unset
 
     // c_cflag - control modes
-    _tio_config.c_cflag |= CS8;  //TODO: Configuration passed by caller
+    _tio_config.c_cflag |= c_cflags;
```

Comment:
Ok, I think we're on the same page here. I'll make those adjustments and push another commit for review.